src/main/java/bisq/network/p2p/network/Connection.java

```diff
@@ -283,22 +283,26 @@ public boolean isCapabilitySupported(NetworkEnvelope networkEnvelop) {
             final PersistableNetworkPayload persistableNetworkPayload = ((AddPersistableNetworkPayloadMessage) networkEnvelop).getPersistableNetworkPayload();
             return !(persistableNetworkPayload instanceof CapabilityRequiringPayload) || isCapabilitySupported((CapabilityRequiringPayload) persistableNetworkPayload);
         } else {
-            return true;
+            return !(networkEnvelop instanceof CapabilityRequiringPayload) || isCapabilitySupported((CapabilityRequiringPayload) networkEnvelop);
```

Comment:
This seems better than before, could there be any bad consequences from not sending some messages that are currently sent?